### PR TITLE
APM/DBM full mode for SQL Server

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do? What is the motivation?

Introducing full mode for APM/DBM for SQL Server.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by lu-zhengda — APPROVED** [review]

**Review by jhgilbert — APPROVED** [review]

**Comment by jhgilbert** [comment]
Thanks, @nenadnoveljic! Just let me know when you would like this merged.

File: content/en/database_monitoring/connect_dbm_and_apm.md [hunk 1]
```diff
@@ -72,6 +72,13 @@ SQL Server and Oracle do not support `full` propagation mode due to statement ca
 
 \* [CommandType.StoredProcedure][25] not supported
 
+\*\* Full mode SQL Server/Java:
+- This mode is in beta.
```
Comment (justiniso) [inline content/en/database_monitoring/connect_dbm_and_apm.md:76]:
```suggestion
```

I don't think this communicates anything. My question as a customer would be "what does beta mean?"

File: content/en/database_monitoring/connect_dbm_and_apm.md [hunk 2]
```diff
@@ -72,6 +72,13 @@ SQL Server and Oracle do not support `full` propagation mode due to statement ca
...
+- The instrumentation makes an additional roundtrip to the database.
```
Comment (justiniso) [inline content/en/database_monitoring/connect_dbm_and_apm.md:77]:
```suggestion
- The instrumentation requires a `SET` query when the client issues a query, which makes an additional round-trip to the database.
```

File: content/en/database_monitoring/connect_dbm_and_apm.md (hunk 1 above)
Comment (nenadnoveljic) [reply content/en/database_monitoring/connect_dbm_and_apm.md:76]: Here `beta` means: "This is a brand new feature. We don't have any known issues, but not many customers are using it.". What would be the best way to put it? Shall we omit `beta` completely?

**Review by justiniso — APPROVED** [review]

File: content/en/database_monitoring/connect_dbm_and_apm.md (hunk 1 above)
Comment (caseycull) [reply content/en/database_monitoring/connect_dbm_and_apm.md:76]: I would remove this line

Comment (nenadnoveljic) [reply content/en/database_monitoring/connect_dbm_and_apm.md]: removed now

**Review by caseycull — APPROVED** [review]

**Comment by nenadnoveljic** [comment]
thx @jhgilbert ! I made a couple of smaller changes based on feedback. Can you take another look an merge if everything OK?

**Review by brett0000FF — APPROVED** [review]

**Pull request merged**